server.go

```diff
@@ -101,9 +100,6 @@ func Serve(opts *ServeConfig) {
 		os.Exit(1)
 	}
 
-	// Logging goes to the original stderr
-	log.SetOutput(os.Stderr)
```

Comment:
I think we still want to do this in case they're not using hclog, right? This shouldn't affect the hclog changes below.